packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -260,6 +355,45 @@ function updateKubeResult() {
           policy may prevent to use hostPort.</span>
       </div>
 
+      <!-- Only show for non-OpenShift deployments (we use routes for OpenShift) -->
+      {#if !openshiftConsoleURL && deployUsingServices}
+        <div class="pt-2 pb-4">
+          <label for="ingress" class="block mb-1 text-sm font-medium text-gray-300"
+            >Expose service locally using Kubernetes Ingress:</label>
+          <input
+            type="checkbox"
+            bind:checked="{createIngress}"
+            name="createIngress"
+            id="createIngress"
+            class=""
+            required />
+          <span class="text-gray-300 text-sm ml-1"
+            >Create a Kubernetes ingress to get access to the exposed ports of this pod. An ingress controller is
```

Comment:
Consider rephrasing:

Create a Kubernetes ingress to get access to the ports that this pod exposes, at the default ingress controller location. Example: on a default Kind cluster created with Podman Desktop: `http://localhost:9090`. Requirements: your cluster has an ingress controller.

Reply:
Updated! :)